_targets.R

```diff
@@ -0,0 +1,52 @@
+library(targets)
+
+tar_option_set(
+  packages = c("dplyr", "readr", "stringr", "sbtools", "whisker"), 
+  error = "stop"
+)
+
+# Could also simplify all the functions to one source file, but following DSP guidelines
+source("1_fetch/src/input_functions.R")
+source("2_process/src/format_functions.R")
+source("3_visualize/src/graph_functions.R")
+
+#exposed parameters
+SBitem <- '5d925066e4b0c4f70d0d0599'
+SBname <- 'me_RMSE.csv'
+outFile1 <- file.path('1_fetch/out',"model_RMSEs.csv")
+
+
+list(
+  #fetch data from ScienceBase
+  tar_target(
+    getData, 
```

Comment:
This is a file target (and all file targets) should be tagged with the file extension type

```suggestion
    getData_csv, 
```